ci/firebase_testlab.sh

```diff
@@ -33,3 +33,16 @@ gcloud --project flutter-infra firebase test android run \
   --results-bucket=gs://flutter_firebase_testlab \
   --results-dir="engine_scenario_test/$GIT_REVISION/$BUILD_ID" \
   --device model=flame,version=29
+
+errors=$(gsutil cat gs://flutter_firebase_testlab/engine_scenario_test/$GIT_REVISION/$BUILD_ID/\*/logcat | grep "[FE]/flutter" | true)
```

Comment:
what does `FE` stand for?  is it Flutter engine?

Reply:
Look for fatal or error warnings, i.e. `F/flutter` or `E/flutter`

Reply:
ah nice! I learned something new about grep